lib/astarte_core/triggers/simple_trigger_config.ex

```diff
@@ -261,6 +275,75 @@ defmodule Astarte.Core.Triggers.SimpleTriggerConfig do
     end
   end
 
+  defp validate_introspection_triggers_interface_version(%Ecto.Changeset{} = changeset) do
+    if Enum.member?(
+         ["interface_minor_updated", "interface_removed", "interface_added"],
+         get_field(changeset, :on)
+       ) do
```

Comment:
This is computed multiple times and the code formatting isn't great. I think it's worth having an helper function for that
```elixir
defp introspection_trigger?(%Ecto.Changeset{} = changeset) do
  Enum.member?(["interface_minor_updated", "interface_removed", "interface_added"], get_field(changeset, :on)
end
```